spec/models/game_spec.rb

```diff
@@ -7,12 +7,4 @@
     new_game.title = older_game.title
     expect(new_game).not_to be_valid
   end
-
-  it 'returns a list of titles' do
-    FactoryGirl.create(:game, title: 'bobGames')
-    FactoryGirl.create(:game, title: 'newBob')
-    FactoryGirl.create(:game, title: 'john')
-    outcome = Game.search('bob')
-    expect(outcome.map(&:title)).to eq(%w(bobGames newBob))
-  end
```

Comment:
Sorry double spaces is ok. I mean to be `specs` in the place of the `specs` you removed.